documentation/airport-guides/katl.md

```diff
@@ -0,0 +1,106 @@
+MCN - Macon Approach
+AHN - Athens Approach
+```
+The airspace available in Openscope is a combination of all ATL positions in the Atlanta sector.
+
+### Airports of Interest
+
+#### KATL (Primary Airport)
+Hartsfield–Jackson Atlanta International  
+881,355 annual operations
+
+Orignally built on an abandoned race track, the Atlanta Speedway, Candler Field was opened in 1926. Already in 1957, the airfield, then Atlanta Municipal Airport, was the busiest airport in the US, with more than two million passengers passing through it, and, between noon and 2 p.m. each day, it became the world's busiest airport. Today, Hartsfield–Jackson is the world's busiest airport by passenger traffic and aircraft operations, handling over 100 million passengers and almsot one million flights yearly. It is the primary hub of Delta Airlines, who operates over 1,000 daily flights, so, to prevent confusion, taxiway Delta is referred to as taxiway Dixie.
+
+##### Instrument Procedures
+In order to help controllers with the the extreme masses of traffic that fly in and out of Atlanta every day, procedures are desgined very carefully and revised regularly to minimize the workload. All procedures for jet aircraft are very automated so a controller's main job is to maintain enough separation between aircraft arriving and departing the area.
+
+##### Runway Configuration
+ATL has 5 parallel runways, the newest one (10/28) opened in 2006. Normally, the inner runways (8R/26L & 9L/27R) are used for departures, while the outer runways (8L/26R & 9R/27L) are used for arrvivals. Since the construction of runway 10/28, triple simultaneous approaches are possible, increasing the capacity of the airport by 40%. This runway can also be used for departures to the east in extremely heavy traffic conditions, if controllers deem it necessary. The tables below show which runways are normally used for all RNAV procedures; ATC may always change the usage to improve traffic flow.
+```
++-----------------+  +--------------------------+
+|       SIDs      |  |   STARs (Landing East)   |
+| 8R/26L | 9L/27R |  |   8L   |   9R   |   10   |
++--------+--------+  +--------+--------+--------+
+| CUTTN2 | BANNG2 |  | CHPPR1 | GLAVN1 | GNDLF2 |
++--------+--------+  +--------+--------+--------+
+| KAJIN2 | GAIRY2 |  | ONDRE1 |        | SITTH2 |
++--------+--------+  +--------+--------+--------+
+| NASSA2 | HAALO2 |  | OZZZI1 |        |        |
++--------+--------+  +--------+--------+--------+
+| PADGT2 | JACCC2 |  +--------------------------+
++--------+--------+  |   STARs (Landing West)   |
+| PENCL2 | PHIIL2 |  |   26R  |   27L  |   28   |
++--------+--------+  +--------+--------+--------+
+| POUNC2 | PLMMR2 |  | CHPPR1 | OZZZI1 | HOBTT2 |
++--------+--------+  +--------+--------+--------+
+| SMKEY2 | SMLTZ2 |  | ONDRE1 |        | JJEDI2 |
++--------+--------+  +--------+--------+--------+
+| VARNM2 | VRSTY2 |  | GLAVN1 |        |        |
++--------+--------+  +--------+--------+--------+
+```
```

Comment:
May be useful to mention the default configuration that Atlanta is running in the sim